docs/api/virtualizer.md

```diff
@@ -80,7 +80,7 @@ The sync parameter indicates whether scrolling is currently in progress. It is `
 overscan?: number
 ```
 
-The number of items to render above and below the visible area. Increasing this number will increase the amount of time it takes to render the virtualizer, but might decrease the likelihood of seeing slow-rendering blank items at the top and bottom of the virtualizer when scrolling.
+The number of items to render above and below the visible area. Increasing this number will increase the amount of time it takes to render the virtualizer, but might decrease the likelihood of seeing slow-rendering blank items at the top and bottom of the virtualizer when scrolling. The default value is `0`.
```

Comment:
The default value is 1, https://github.com/TanStack/virtual/blob/main/packages/virtual-core/src/index.ts#L422

```suggestion
The number of items to render above and below the visible area. Increasing this number will increase the amount of time it takes to render the virtualizer, but might decrease the likelihood of seeing slow-rendering blank items at the top and bottom of the virtualizer when scrolling. The default value is `1`.
```